test/validation/api/classification/odp_classification_common.c

```diff
@@ -558,11 +558,28 @@ odp_cls_pmr_term_t find_first_supported_l3_pmr(void)
 	else if (capability.supported_terms.bit.tcp_dport)
 		term = ODP_PMR_TCP_DPORT;
 	else
-		CU_FAIL("Implementations doesn't support any TCP/UDP PMR");
+		CU_FAIL("Implementation doesn't support any TCP/UDP PMR");
 
 	return term;
 }
 
+cls_packet_l4_info find_first_supported_proto(void)
+{
+	odp_cls_capability_t capability;
+
+	odp_cls_capability(&capability);
```

Comment:
If this fails, the conditions below are testing potentially unitialized data.

Reply:
Fixed in v2. Fixed also similar problems in the existing code.